scripts/populate_institutions.py

```diff
@@ -71,6 +72,7 @@ def main(env):
                 'logout_url': SHIBBOLETH_SP_LOGOUT.format(encode_uri_component('https://osf.io/goodbye')),
                 'domains': [],
                 'email_domains': [],
+                'delegation_protocol': 'SAML_SHIBBOLETH',
```

Comment:
I would just make sure that you make clear the difference in the choices tuple referenced in the comment above.